### PR TITLE
Added zipkin and js client

### PR DESCRIPTION
**Pull request by anamonium**

### CODE REVIEW

File: app/server.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import express from 'express';
+import fetch from 'isomorphic-fetch';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import { DaprClient, HttpMethod } from "@dapr/dapr";
+
+const app = express();
+app.use(express.json());
+
+const port = process.env.APP_PORT ?? '3000';
```
Comment (strjakub) [inline app/server.js]: I know it's a convention but in our use case we want the port to match dapr command, so setting it up to the value used in README should be enough

File: app/server.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+const serviceAppId = "server";
+const serviceMethod = "health";
+
+
+app.get('/health', async (_req, res) => {
+    const response = await client.invoker.invoke(serviceAppId, serviceMethod, HttpMethod.GET);
+    console.log(response);
+    res.json(response);
+});
```
Comment (strjakub) [inline app/server.js:29]: I set up this endpoint to see if app is alive. In the near future we should split it up on java server in two separate where on is just a pong and the other with more appropriate name generates ID, do business logic and return it. You don't need to do it right now if you don't want to, just sayin'.

File: app/server.js (hunk 1 above)
Comment (anamonium) [reply app/server.js]: Oki, will update.

File: app/server.js (hunk 2 above)
Comment (anamonium) [reply app/server.js:29]: Same here. Wanted to be sure that the client-server in js communicates well with an actual java server.

File: app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Express App</title>
+</head>
+<body>
+    <h1>Client</h1>
+    <button id="getHealthButton">Get health</button>
+    <script src="/client.js"></script>
```
Comment (strjakub) [inline app/public/index.html]: That script doesn't impact the render of the page and is not a performance issue, so it should be put in head tags and not inside the body.

File: README.md [hunk 4]
```diff
@@ -16,3 +22,12 @@ dapr run --app-id receiverLog --app-protocol http --app-port 8002 --dapr-http-po
 ```
 node app.js
 ```
+
+### Mini klient + server js (mozliwe ze sie w portach zagubilam ;) )
```
Comment (strjakub) [inline README.md]: If it works then you did not xd, maybe it is wise to leave that digression off the main branch xD

File: app/server.js (hunk 2 above)
Comment (strjakub) [reply app/server.js:29]: so leave it, I'll do it someday

**Review by strjakub — CHANGES_REQUESTED** [review]
.

**Review by strjakub — APPROVED** [review]

**Pull request merged**